MaxSukhanov/school_docs
# create_user_model_and_configure_email_confirmations

**What's new here?**

Now User can register or log in. 

**How to check it out?**

0. Launch mailcatcher to get a confirmation email: `$ mailcatcher`
1. Go to `http://localhost:3000/users/sign_up`
2. Create a dummy account
3. Try to log in to make sure that access restricted until account is confirmed 
4. Open mailcatcher, confirm account by the link from email
5. Log in successfully
